DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/part2_lib_func_call/src/use_library.cpp

```diff
@@ -0,0 +1,113 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <sycl/ext/intel/ac_types/ac_int.hpp>
+// oneAPI headers
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/ext/intel/experimental/fpga_kernel_properties.hpp>
+#include <sycl/sycl.hpp>
+
+#include "lib_rtl.hpp"
+
+#include "exception_handler.hpp"
+
+// Forward declare the kernel name in the global scope.
+// This FPGA best practice reduces name mangling in the optimization report.
+class KernelComputeRTL;
+
+using MyInt27 = ac_int<27, false>;
+using MyInt54 = ac_int<54, false>;
+
+
+// Using host pipes to stream data in and out of kernal
+// IDPipeA and IDPipeB will be written at host, and then read data in kernel (device)
+// IDPipeC will be written at kernel (device) respectively, and then read data by host
+class IDPipeA;
+using InputPipeA = sycl::ext::intel::experimental::pipe<IDPipeA, unsigned>;
+class IDPipeB;
+using InputPipeB = sycl::ext::intel::experimental::pipe<IDPipeB, unsigned>;
+class IDPipeC;
+using OutputPipeC = sycl::ext::intel::experimental::pipe<IDPipeC, unsigned long>;
+
+// This kernel compute multiplier result by call RTL function RtlDSPm27x27u
+template <typename PipeIn1, typename PipeIn2, typename PipeOut>
+struct RtlMult27x27 {
+
+  auto get(sycl::ext::oneapi::experimental::properties_tag) {
+    return sycl::ext::oneapi::experimental::properties{
+        sycl::ext::intel::experimental::streaming_interface_accept_downstream_stall, 
+        sycl::ext::intel::experimental::pipelined<1>};
+  }
+  
+  void operator()() const {
+    unsigned a_val = PipeIn1::read();
+    unsigned b_val = PipeIn2::read();
+    MyInt27 a = a_val;
+    MyInt27 b = b_val;
+    MyInt54 res = RtlDSPm27x27u(a, b);
```

Comment:
```suggestion
    MyInt27 a_val = PipeIn1::read();
    MyInt27 b_val = PipeIn2::read();
    MyInt54 res = RtlDSPm27x27u(a_val, b_val);
```
be consistent with the use_native.cpp